pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -410,7 +410,7 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 // TODO: replace with CNI traffic shaper plugin
 func (plugin *kubenetNetworkPlugin) addTrafficShaping(id kubecontainer.ContainerID, annotations map[string]string) error {
 	shaper := plugin.shaper()
-	ingress, egress, err := bandwidth.ExtractPodBandwidthResources(annotations)
+	ingress, _, egress, _, err := bandwidth.ExtractPodBandwidthResources(annotations)
```

Comment:
Could you add here a `TODO` since these values should be used somewhere in the future (we can resolve this in an additional PR)